.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+# Description
+
+Fixes #[issue number]
+[Please include a summary of the changes and the related issue]
+
+## Demo
+
+[If possible, please include a screenshot or gif/video, it'll make it easier for reviewers to understand the scope of the changes and how the change is supposed to work. If you're introducing something new or changing the existing patterns, please share a Loom and explain what decisions you've made and under what circumstances]
+
+## Release notes
+
+[Include a human-readable description that could help to prepare a changelog]
+
+## Testing scenarios
+
+[Please list all the testing scenarios a reviewer has to check before approving the PR]
+
+- [ ] Scenario A
+  - Step 1
+  - Step 2...
+
+- [ ] Scenario B
+  - Step 1
+  - Step 2....
+
+## Final checklist
+
+- [ ] I checked the [code review guidelines](../../docs/codeReview.md)
+- [ ] I have performed a self-review of my code, the same way I'd do it for any other team member
+- [ ] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
+- [ ] I have commented on my code, particularly in hard-to-understand areas
```

Comment:
+1 I have some reservations listing it like this, maybe change it to
> I added a comment when I took a non-obvious choice why I did it this way

Reply:
what about 
> Whenever I took a non-obvious choice I added a comment explaining why I did it this way 
?